lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/app-doc-check.yaml

```diff
@@ -0,0 +1,69 @@
+name: DCMAW and Driving Licence
+description: >-
+  The combined journey for DCMAW and Driving Licence CRIs.
+entryEvents:
+  next:
+    targetState: CRI_DCMAW
+nestedJourneyStates:
+  CRI_DCMAW:
+    response:
+      type: cri
+      criId: dcmaw
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE
+      dl-auth-source-check:
+        targetState: CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK
+      not-found:
+        exitEventToEmit: incomplete-not-found
+      access-denied:
+        exitEventToEmit: incomplete
+      temporarily-unavailable:
+        exitEventToEmit: incomplete
+      fail-with-no-ci:
+        exitEventToEmit: incomplete
+      enhanced-verification:
+        exitEventToEmit: incomplete
+
+  CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK:
+    response:
+      type: cri
+      criId: drivingLicence
+      context: check_details
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE
+      access-denied:
+        targetState: RESET_SESSION_BEFORE_PROVE_ANOTHER_WAY_AFTER_DL
+      alternate-doc-invalid-dl:
+        targetState: RESET_SESSION_BEFORE_DL_ALT_DOC_MITIGATION
+
+  RESET_SESSION_BEFORE_PROVE_ANOTHER_WAY_AFTER_DL:
+    response:
+      type: process
+      lambda: reset-session-identity
+      lambdaInput:
+        resetType: ALL
```

Comment:
(Don't worry about this now - but when we add the escape route we might need to change this so it only resets DCMAW, not everything)